docs/rules/no-useless-call.md

```diff
@@ -42,6 +42,16 @@ foo.apply(null, args);
 obj.foo.apply(obj, args);
 ```
 
+Known limitations:
+
+```js
+// This is warned.
+a[i++].foo.call(a[i++], 1, 2, 3);
+
+// This is not warned.
+a[++i].foo.call(a[i], 1, 2, 3);
```

Comment:
Maybe you could add a little more explanation here?  It may not be immediately obvious what the real limitation is.  Is the limitation only that specific example (`++i` and `i`), or is it more general?